File: Java/11.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
class sumof2no
{
    public static void main(String[] args) throws IOException //Handles IO Exception errors.
    {
        int a,b,sum;
        BufferedReader br = new BufferedReader (new InputStreamReader(System.in)); //Create a new object 'br' of class 'BufferedReader'.
        System.out.print("Enter two numbers:");
        a = Integer.parseInt(br.readLine()); //Take 1st number as input.
        b = Integer.parseInt(br.readLine()); //Take 2nd number as input.
        sum = a + b; //Calculate the sum.
        System.out.println("Sum: "+sum); //Print the sum.
    }
}

File: Python/11.py
c = 0
p = 1.0
a = int(input('Enter the total number of elements: '))
while c<a:
    x = int(input('Enter the elements: '))
    c = c + 1
    p = p * x
gm = p ** (1.0/a)
print('The Geometric Mean is: ',gm)